src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,17 @@
+.catalog {
+  display: flex;
+  box-sizing: border-box;
+  flex-wrap: wrap;
+  margin: 0 auto;
+  padding: $catalog-padding $catalog-padding 0;
```

Comment:
Let's fix the tests
![image](https://user-images.githubusercontent.com/99331793/198580398-59a69c48-c3eb-430c-9c4d-8a6fb1b0dd33.png)
![image](https://user-images.githubusercontent.com/99331793/198580455-c1ac73dd-f99a-41e4-a42e-a4d1cb81f5f2.png)

![image](https://user-images.githubusercontent.com/99331793/198580633-82b86cc7-af4b-4934-8034-069d1bb78762.png)